unicodetools/org/unicode/props/ExtraPropertyValueAliases.txt

```diff
@@ -145,3 +145,5 @@ sc ; Zmth                             ; Math_Symbols
 sc ; Zsye                             ; Emoji_Symbols
 sc ; Zsym                             ; Other_Symbols
 sc ; Zxxx                             ; Unwritten
+
+age; 13.1                             ; V13_1
```

Comment:
Please add a TODO pointing to the new bug/issue. Once the code is fixed, we should not need a fake Age property value.

Optional: This file should be sorted, like the real thing in the UCD. I would move this line to near the top.

Reply:
```suggestion
# TODO: There is no Unicode 13.1
age; 13.1                             ; V13_1
```

Reply:
Please add bug/issue links to these TODOs.

Reply:
I'm still not quite sure I understand what/if there is a problem. But I'll open something for 'revisit 13.1' and link it in.

Reply:
For the UCD Age property, there is no 13.1=V13_1 value. This file adds stuff that the tools need but the UCD doesn't have, and maybe we need this value to process *emoji* 13.1, but we should look at that.

Reply:
suggested change lg